src/www/templates/committees/committees.html

```diff
@@ -0,0 +1,31 @@
+<div class="container">
+
+    <!-- TODO: Abstract from container div like pages -->
+
+    <h1>Committees</h1>
+    <p>
+        The association has several committees that focus on organizing activities, workshops, events and trips.
+        In addition, several committees help the association in acquisition and promotion.
+        In total, H.S.A. Confluente currently consists of {{committees.length}} committees.
```

Comment:
Does Confluente only consist of its committees? Would change it to 'is currently supported by' or something